reglemente/sektionskomitteer/sektionskomitteer.tex

```diff
@@ -16,7 +16,8 @@ \subsection{Förteckning}
 \MRCIT       & 2 - 5                  & LP2     & 1/1       \\ \hline
 \NOLLKIT     & 2 - 5                  & LP2     & 1/1       \\ \hline
 \PRIT        & 2 - 6                  & LP2     & 1/1       \\ \hline
-\SEXIT       & 3 - 6                  & LP2     & 1/1                                         
+\SEXIT       & 3 - 6                  & LP2     & 1/1       \\ \hline
+\EQUALIT     & 0 - 2                  & LP4     & 1/7       \\ \hline
```

Comment:
should there be a hline on the last one?

Reply:
Ah, it was not intentional